target/linux/ramips/dts/mt7621_thunder_timecloud.dts

```diff
@@ -95,10 +95,19 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0xe000>;
 };
 
+&switch0 {
+	ports {
+		port@4 {
```

Comment:
So, for this device you know it's port 4, though it has the same definition as mikrotik,rbm11g?

Reply:
I found that it's port 4 from some sources
https://www.right.com.cn/FORUM/thread-269337-1-1.html

Reply:
Fine.